bin/release

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+# shellcheck source=functions
+if [[ "$(basename "$0")" == release ]] ; then
+  source "$(dirname "$0")/../functions"
+else
+  source "$(dirname "$0")/../sanctuary-scripts/functions"
+fi
+
+declare -r increment="$1"
+owner="$(get repo-owner)"
+name="$(get repo-name)"
+prefix="$(get version-tag-prefix)"
+
+node_modules/.bin/xyz \
+  --increment "${increment}" \
+  --repo "git@github.com:${owner}/${name}.git" \
+  --script scripts/prepublish \
```

Comment:
Now that there's way to configure some of these scripts, should this field remain hard-coded? Maybe

```config
  prepublish-script = scripts/prepublish
```

Allowing it to be emptied, similarly to the `version-prefix` option?

Reply:
I decided not to make this change after all, as we're already relying on convention for overriding the behaviour of scripts. To override `sanctuary-lint`, for example, one would create an executable at __scripts/lint__.

Note that this pull request includes a default prepublish script suitable for use in most Sanctuary projects.